Dockerfile

```diff
@@ -9,8 +9,7 @@ RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -trimpath -o /usr/b
 FROM alpine:3.18 as release
 
 # hadolint ignore=DL3018
-RUN apk add --no-cache ca-certificates && \
-    adduser -D -u 12345 -g 12345 k6
+RUN adduser -D -u 12345 -g 12345 k6
```

Comment:
I'm not sure if it's required to remove the `ca-certificates` :man_shrugging:  From my point of view, `Dockerfile` is a different "layer" and could stay as it's untouched. There is no cost of maintaining it.

However, if we decide to remove it, let's also clear the comment `hadolint ignore=DL3018` since it relates to that package.

Reply:
>Dockerfile is a different "layer"

It is but they have the same end goal.

>There is no cost of maintaining it.

From my point of view, it has a cost in case we need to troubleshoot an issue. It could be difficult to be sure what was the used certificate from the system.

@mem I share @olegbespalov's request. Can you drop the `hadolint` line, please?